tools/shared.py

```diff
@@ -2908,12 +2908,23 @@ def run_binaryen_command(tool, infile, outfile=None, args=[], debug=False, stdou
       cmd += ['--input-source-map=' + infile + '.map']
       cmd += ['--output-source-map=' + outfile + '.map']
 
-    return run_process(cmd, stdout=stdout).stdout
+    ret = run_process(cmd, stdout=stdout).stdout
+    if outfile:
+      Building.debug_copy(outfile, 'rbc.%s.wasm' % tool)
```

Comment:
Can you stop the `rbc` that seems like it just adds noise to the output filename?